### ISSUE TITLE
SelectSql в TIBDataSet  не понимает CTE

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#3324]
Originally reported on Google Code with ID 3324

```
Если в SelectSql  TIBDataSet-а передать текст запроса содержащего: “With cte as ( …”,
то выдаётся ошибка.

```

Reported by `igor6003` on 2014-03-05 08:47:47

**Comment by gsbelarus** [comment]
```
Какая ошибка?
```

Reported by `gs1994` on 2014-03-05 12:47:46

**Comment by gsbelarus** [comment]
```
Ошибка что неправильный символ ‘)’. Т.е.  из части  “With cte as (…)” выкидывается всё
кроме последней закрывающей скобки.
```

Reported by `igor6003` on 2014-03-05 12:55:03

**Comment by gsbelarus** [comment]
```
Дайте полностью весь запрос.
```

Reported by `gs1994` on 2014-03-05 12:55:54
- **Labels added**: Type-Defect, Priority-High, GedeminEXE

**Comment by gsbelarus** [comment]
```
Он есть в Issue 2574, в конце.
```

Reported by `igor6003` on 2014-03-05 13:03:01

**Comment by gsbelarus** [comment]
```
У вас точно последняя версия экзешника? У меня нормально проходит. Проверьте:

1) вставьте запрос в окно SQL редактора
2) нажмите кнопку подготовить
3) нажмите кнопку Разобрать запрос
4) нажмите кнопку Подготовить

у меня не кидает ошибок. 
```

Reported by `gs1994` on 2014-03-05 13:07:35

**Comment by gsbelarus** [comment]
```
В SQL редакторе всё нормально.
А в ф-ции function acc_scr_CalcSaldo делаю останов в конце на qry.Open смотрю qry.SelectSQL.Text
– там запрос правильный и он работает. Если делаю продолжить по F7, то получаю выше
описанную ошибку. 
Экзешник 2.5.19.8101.

```

Reported by `igor6003` on 2014-03-05 13:26:48

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2014-03-05 14:56:33
- **Status changed**: `Duplicate`
- **Merged into**: #3325
